aiohttp/web_reqrep.py

```diff
@@ -164,12 +164,21 @@ def _splitted_path(self):
         return urlsplit(self._path_qs)
 
     @property
+    def unquoted_path(self):
```

Comment:
Good catch. IIRC I've suggested `unquoted_path` name but `raw_path` is definitely better name.

@kxepal thank you again for advice.

You know, naming and cache invalidation are the most hard things in computer science :)